drivers/storage/portworx/component/telemetry.go

```diff
@@ -683,42 +692,50 @@ func (t *telemetry) createCCMGoConfigMapCollectorProxyV2(
 	cluster *corev1.StorageCluster,
 	ownerRef *metav1.OwnerReference,
 ) error {
-	configMap := getCollectorProxyConfigMapV1(cluster, ownerRef)
-	configMap.Name = ConfigMapNameTelemetryCollectorProxyV2
-	proxy := pxutil.GetPxProxyEnvVarValue(cluster)
-	if proxy == "" || !t.usePxProxy {
-		// TODO: use offset in the configmap as well
-		return k8sutil.CreateOrUpdateConfigMap(t.k8sClient, configMap, ownerRef)
-	}
-
-	address, port, err := pxutil.SplitPxProxyHostPort(proxy)
-	if err != nil {
-		logrus.Errorf("failed to get custom proxy address and port from %s: %v", proxy, err)
-		return k8sutil.DeleteConfigMap(t.k8sClient, ConfigMapNameTelemetryCollectorV2, cluster.Namespace, *ownerRef)
-	}
-
 	// TODO: share same config template with phonehome, component-sn field is needed here
-	configFileName := configFileNameTelemetryCollectorCustomProxy
+	configFileName := configFileNameTelemetryCollectorProxy
 	cloudSupportPort, tcpProxyPort, envoyRedirectPort := getCCMCloudSupportPorts(cluster, defaultCollectorPort)
 	replaceMap := map[string]string{
-		configParameterApplianceID:                           cluster.Status.ClusterUID,
-		configParameterComponentSN:                           "portworx-metrics-node",
-		configParameterProductVersion:                        pxutil.GetPortworxVersion(cluster).String(),
-		configParameterRestProxyURL:                          getArcusRestProxyURL(cluster),
-		configParameterCloudSupportPort:                      fmt.Sprint(cloudSupportPort),
-		configParameterCloudSupportTCPProxyPort:              fmt.Sprint(tcpProxyPort),
-		configParameterCloudSupportEnvoyInternalRedirectPort: fmt.Sprint(envoyRedirectPort),
-		configParameterCustomProxyAddress:                    address,
-		configParameterCustomProxyPort:                       port,
+		configParameterApplianceID:          cluster.Status.ClusterUID,
+		configParameterProductVersion:       pxutil.GetPortworxVersion(cluster).String(),
+		configParameterRestProxyURL:         getArcusRestProxyURL(cluster),
+		configParameterRestCloudSupportPort: fmt.Sprint(cloudSupportPort),
+		configParameterComponentSN:          "portworx-metrics-node",
+	}
+
+	proxy := pxutil.GetPxProxyEnvVarValue(cluster)
+	if proxy != "" && t.usePxProxy {
+		address, port, err := pxutil.SplitPxProxyHostPort(proxy)
```

Comment:
should log error instead of info at this line https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/util/util.go#L593

Reply:
oh that was for debugging, will delete that line.